exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
+    test:
```

Comment:
There should be an `external-service` entry here, which uses the external Docker image.